standard/json.lua

```diff
@@ -0,0 +1,38 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Json
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local json = {}
+
+local getArgs = require('Module:Arguments').getArgs
+
+function json.fromArgs(frame)
+	local args = getArgs(frame)
+	return json.stringify(args)
+end
+
+function json.stringify(obj, pretty)
+	return mw.text.jsonEncode(obj, pretty == true and mw.text.JSON_PRETTY or nil)
+end
+
+function json.parse(obj)
+	local parse = function(object) return mw.text.jsonDecode(object, mw.text.JSON_TRY_FIXING) end
+	local status, res = pcall(parse, obj);
+	if status then
+		return res, false
```

Comment:
What does this boolean value mean? Why are we returning a tuple?

Reply:
idk it is just a copy from live
i assume Quadrat had a reason